### DISCUSSION TITLE
Video returned by YouTube isn't what was requested

### DISCUSSION THREAD

**Discussion by CripperFunny666** [jagrosh/MusicBot#1540] [Troubleshooting]
If they try to start a video through the search command, the bot will exit the channel. How to fix it?
![Снимок экрана 2024-04-10 220725](https://github.com/jagrosh/MusicBot/assets/153866760/3ba89f9a-c35e-4311-98e8-3f2b2d4cc648)

**Comment by jagrosh** [comment]
Duplicate of #1527
